kuma/wiki/tests/test_feeds.py

```diff
@@ -467,7 +467,7 @@ def test_feeds_update_after_doc_tag_change(client, wiki_user, root_doc):
         response = client.get(reverse('wiki.feeds.recent_documents',
                                       args=['atom', tag]), follow=True)
         assert response.status_code == 200
-        assert root_doc.title not in response.content
+        assert root_doc.title not in response.content.decode()
```

Comment:
nit: Don't omit the encoding in general. The docs say this uses the "default string encoding" when omitted.  ``sys.getdefaultencoding()`` returns ``ascii`` for me. Probably OK for the test, if there are no non-ASCII characters in the response, but not in general.